lib/Cleantalk/ApbctWP/FindSpam/UsersChecker.php

```diff
@@ -491,17 +527,18 @@ public static function ctGetCsvFile()
             if ( is_array($user_meta) ) {
                 $user_meta = array_values($user_meta);
             }
+            $ip_of_user_meta = TT::getArrayValueAsString($user_meta, 'ip');
```

Comment:
A bug is here: `TT::getArrayValueAsString` here will find `$user_meta['ip']` but not `$user_meta[0]['ip']` needed.